docs/zh_cn/_static/css/readthedocs.css

```diff
@@ -1,6 +1,62 @@
 .header-logo {
-    background-image: url("../image/lmdeploy-logo.png");
-    background-size: 150px 60px;
+    background-image: url("../image/lmdeploy-logo.svg");
```

Comment:
There's a wrong url, could fix in this pr:
https://github.com/InternLM/lmdeploy/blob/e7eedc0384a5bf4201d75f7b00f85c2bc60f5a3f/docs/zh_cn/conf.py#L112

Reply:
fixed